lib/internal/per_context/domexception.js

```diff
@@ -2,14 +2,23 @@
 
 const {
   Error,
+  FunctionPrototypeCall,
   ObjectDefineProperties,
   ObjectDefineProperty,
-  SafeWeakMap,
+  ObjectGetOwnPropertyDescriptors,
+  ObjectGetPrototypeOf,
+  ObjectSetPrototypeOf,
+  ReflectConstruct,
   SafeMap,
+  SafeWeakMap,
+  SymbolFor,
   SymbolToStringTag,
   TypeError,
 } = primordials;
 
+const kClone = SymbolFor('nodejs.clone');
```

Comment:
These seems to be a bit too general, I can imagine that other APIs might want to provide cloning or deserialization customization in the future too. Perhaps we can use the `nodejs.worker_threads.` prefix to be a bit more specific (not sure if `worker_threads` is the best namespace though, but it is a namespace that already exists)

Reply:
Yes, `worker_threads` sounds good. Added in https://github.com/nodejs/node/pull/41148/commits/012d03bfe5bf122755af1446500ddf1fd54e27e1.